### DISCUSSION TITLE
Ability to define separate dark & light mode logos

### DISCUSSION THREAD

**Discussion by fosron** [laravel/nova-issues#5246] [Ideas & Feature Requests]
Hey,

it would be cool if we could define separate logo SVGs for dark & light modes. We are aware that its possible to have a single SVG for different modes, but it feels hacky, and a proper first party solution would be good.

WDYT?

Thanks.

**Comment by pedro-santiago** [comment]
If you use the classes `dark:` inside your SVG, it does not solve the problem?
